_deps/mod.ts

```diff
@@ -0,0 +1,30 @@
+// "@polkadot/api": "^8.9.1",
```

Comment:
Hi @harrysolovay I'm not sure the purpose of this file? is something deno related?

Reply:
It's a temporary file that lists out all deps, both dev & runtime. It's useful for us to gradually delete items as we reference the Deno-compatible versions / replacements.